packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -122,6 +122,13 @@ export interface PluginConfigTypeAndroid {
    * @see [Android documentation](https://developer.android.com/guide/topics/manifest/application-element#usesCleartextTraffic)
    */
   usesCleartextTraffic?: boolean;
+  /**
+   * Specifies the set of other apps that an app intends to interact with. These other apps are specified by package name,
+   * by intent signature, or by provider authority.
+   *
+   *  @see [Android documentation](https://developer.android.com/guide/topics/manifest/queries-element)
+   */
+  queries?: PluginConfigTypeAndroidQueries;
```

Comment:
```suggestion
  manifestQueries?: PluginConfigTypeAndroidQueries;
```

since the android section covers settings from build.gradle, AndroidManifest.xml, and even proguard rules, would be good to give hint that the queries are AndroidManifest's queries.